modules/gitops-release-notes-1-3-2.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-2_{context}"]
+= Release notes for {gitops-title} 1.3.2
+
+{gitops-title} 1.3.2 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
+
+[id="new-features-1-3-2_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.2:
+
+* Upgraded Argo CD to version *v2.1.8*
+
+* Upgraded Dex to version *v2.30.0*
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+The following issues were resolved in the current release:
+
+* Previously, in the `OperaturHub` if you filter by "Disconnected" under the *Infrastructure Features* the {gitops-title} did not appear, since the Operator did not had the related annotation set in its CSV. The {gitops-title} is now annotated with "Disconnected Cluster" as an *Infrastructure Feature*. link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
+
+* When using an "Namespace-scoped" argocd (e.g. one that is not scoped to All Namepsaces in a cluster), the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces includes the `argocd.argoproj.io/managed-by` label. This list of namespaces gets cached in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. However, if one of these namespaces gets deleted, the operator doesn't seem to notice, and the namespace remains in that list. This breaks both the *CONNECTION STATE* in that cluster config, and also all sync attempts after the namespace gets deleted. The sync will show errors. For example:
```

Comment:
```suggestion
* When using an "Namespace-scoped" argocd, for example, argocd that is not scoped to All Namepsaces in a cluster, the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces include the `argocd.argoproj.io/managed-by` label. This list of namespaces is stored in a cache in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. Before this update, if you deleted one of these namespaces, the Operator ignored that, and the namespace remained in the list. This behavior broke the *CONNECTION STATE* in that cluster configuration, and all sync attempts resulted in errors. For example:
```
Most suggestions are based on the IBM SG. We should avoid using Latin abbreviations (e.g.) and using "cache" as a verb. Gets deleted is a bit colloquial. And we always describe previous problems with the past tenses.